src/development/tools/devtools/vscode.md

```diff
@@ -18,28 +18,24 @@ in VS Code and clicking **Run > Start Debugging** (`F5`).
 ## Launch DevTools
 
 Once the debug session is active and the application has started,
-the **Dart: Open DevTools** command becomes available in the
-VS Code command palette:
+the **Open DevTools** commands become available in the
+VS Code command palette (`F1`):
 
-![Screenshot showing Open DevTools command]({{site.url}}/assets/images/docs/tools/vs-code/vscode_command.png){:width="100%"}
+![Screenshot showing Open DevTools commands]({{site.url}}/assets/images/docs/tools/vs-code/vscode_command.png){:width="100%"}
 
-The first time you run this (and subsequently when the DevTools package
-is updated), you are prompted to activate or upgrade DevTools.
+The chosen tool will be opened embedded inside VS Code.
 
-![Screenshot showing Active DevTools command]({{site.url}}/assets/images/docs/tools/vs-code/vscode_install_prompt.png){:width="100%"}
+![Screenshot showing DevTools embedded in VS Code]({{site.url}}/assets/images/docs/tools/vs-code/vscode_embedded.png){:width="100%"}
 
-Clicking the **Open** button uses `pub global activate` to activate
-the DevTools package for you. Next, DevTools launches in your browser and
-automatically connects to your debug session.
+You can choose to have DevTools always opened in a browser with the
+`dart.embedDevTools` setting, and control whether it opens as a full window or
+in a new column next to your current editor with the `dart.devToolsLocation`
+setting.
```

Comment:
It would be great to link to [the recommended settings](https://dartcode.org/docs/recommended-settings/) for context here.

Reply:
I can add this, although I'm not sure how to frame it because it's not really related do this - none of those settings are in the recommended settings (because the recommended settings are the defaults). The recommend settings are really things that aren't Dart-specific and/or might be surprising if we configured them by default.

We could link to the [full settings list](https://dartcode.org/docs/settings/), something like:

> A full list of Dart/Flutter settings are available [here](https://dartcode.org/docs/settings/), along with some recommendation settings for Dart/Flutter in VS Code [here](https://dartcode.org/docs/recommended-settings/).

Although I don't know if for the general settings it'd be better to direct users to just browse them in VS Code (the settings editor has some groupings/filters and a search box).

Reply:
When I read this, I wasn't sure what a setting was, so I googled "Dart VSCode settings" or something. Any of those options sounds good to me.